local_peer.go

```diff
@@ -8,13 +8,20 @@ import (
 	"time"
 )
 
+const (
+	topologyUpdateInterval = 1 * time.Second
```

Comment:
"update interval" isn't quite how I think about this.
What about "defer duration" ?